docs/04_Run_changes_in_App.md

```diff
@@ -22,14 +22,14 @@ sometimes have to adjust the udev rules in order to
 
 ### Run Your Changes on the Extractor
```

Comment:
Why bother with JitPack when you only need the commit number:
As said, you need to push online, github or gitlab. but then, you can use ``git log`` to copy the commit number, there is no need to deal with JitPack, it's way easier and faster. Then I think we should rewrite this section.

Reply:
Yes, we might want to add a config for local development, too. I guess most devs do not like to wait for JitPack to build their changes, to be able to test them in the app. @Stypox Since you are working on the Gradle file, can you please look into this?

Reply:
You can use your modified version of Extractor without push? I'm not talking about the app here.
If yes, we should definitely talk about that

Reply:
@TobiGr I never tried to link NewPipe to a local NewPipeExtractor build, so I don't know which method should be used.
The first thing that comes to my mind is creating a jar from the Extractor and adding it as a library in NewPipe, but I'm not sure if there are simpler methods

Reply:
Keep in mind JitPack is also proprietary software (at least the server side), and we should avoid adding dependencies on those.
It's much harder to move away from such a service than to move to another Git hosting system for instance, and even if Travis broke or we would move away from it, the local builds never depend on Travis.

Reply:
See TeamNewPipe/NewPipe#2535 for one of the ways to do it with a local build of NewPipeExtractor